stdlib/Pkg3/README.md

```diff
@@ -36,14 +36,14 @@ INFO: Resolving package versions
 INFO: Installing Compat v0.28.0 [0427af7e2c8610c8]
 INFO: Updating project file /Users/stefan/projects/julia/Project.toml
 INFO:  [+] Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
-INFO: Updating manifest file /Users/stefan/projects/julia/Manifest.toml
+INFO: Updating lockfile file /Users/stefan/projects/julia/Lockfile.toml
 INFO:  [34da2185] ↓ Compat v0.31.0 ⇒ v0.28.0
 
 pkg> rm Compat
 INFO: Updating project file /Users/stefan/projects/julia/Project.toml
 INFO:  [-] Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
 INFO:  [-] JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
-INFO: Updating manifest file /Users/stefan/projects/julia/Manifest.toml
+INFO: Updating lockfile file /Users/stefan/projects/julia/Lockfile.toml
```

Comment:
"lockfile file"?

Reply:
It's just a straw man PR to see how we feel about it.